modules/nw-configuring-router-compression.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+[id="nw-configuring-router-compression_{context}"]
+= Using router compression
+
+You configure the HAProxy Ingress Controller to specify router compression. You can use the `mimeTypes` variable to define the formats of MIME types to which compression is applied. The types are: application, image, message, multipart, text, video, or a custom type prefaced by "X-". To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341].
+
+[NOTE]
+====
+Memory allocated for compression can affect the max connections. Additionally, compression of large buffers can cause latency, like heavy regex or long lists of regex.
```

Comment:
I got this Note info from a comment in NE-542. I feel like a little more info is needed around "max connections". Wdyt @candita ?

Reply:
like max connections for ingress operator?

Reply:
That comment is a little out of date, but the idea is important, thanks for your diligence on this. I suggest:

If the HAProxy doesn't have enough resources to perform compression, it may skip the compression in order to stay functional.

There is also a useful `note` in https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L257-L261